packages/containers/src/Slider/Slider.container.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { componentState, cmfConnect } from '@talend/react-cmf';
+import { Slider as Component } from '@talend/react-components';
+import omit from 'lodash/omit';
+import Immutable from 'immutable';
+import PropTypes from 'prop-types';
+
+export const VALUE_ATTR = 'value';
+export const DEFAULT_STATE = new Immutable.Map({
+	[VALUE_ATTR]: undefined,
+});
+
+export const DISPLAY_NAME = 'Container(Slider)';
```

Comment:
This const is only used in the container displayName. Why do you export it ?

Reply:
used in the test file also